clappr/clappr-android
# Feature/init destroy plugin

### Goal
Add stop listeners to container and playback listeners

### How to Test
1 - Run Unit Tests
2 - Open sample app
3 - Check if media control plugins are appearing in the video
